composer.json

```diff
@@ -15,6 +15,7 @@
     ],
     "require": {
         "php": "^5.3.6 || >=7.0 <7.2",
+        "doctrine/annotations": "^1.2",
```

Comment:
My point was not that we couldn't install it, it was that newer versions will not support old php, and then we'll have no choice but to upgrade when ever there are a mountain of conflicts with other people wanting a 2.x version of this package, but we need 1.x.